manifests/l/LegacyInteractive/Launcher/1.12.0/LegacyInteractive.Launcher.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: LegacyInteractive.Launcher
+PackageVersion: 1.12.0
+PackageLocale: en-US
+Publisher: Legacy Interactive, Inc.

```

```suggestion
Publisher: Legacy Games
```
This should match control panel